uber/decorators.py

```diff
@@ -485,3 +485,20 @@ def __call__(self, func):
         redirect_func = create_redirect(self.url or '../' + get_module_name(func) + '/' + func.__name__)
         setattr(root, self.alias_name or func.__name__, redirect_func)
         return func
+
+
+def id_required(func):
```

Comment:
Since this ONLY checks for Attendee id, we should probably rename this decorator ``attendee_id_required``.  Otherwise someone might think they could use it for other functions with an ``id`` parameter which get a ``Job`` or something.

A later improvement could be to make this function smarter so that it checks ``func.__module__`` to see which model to query, e.g. all of the functions in ``registration.py`` and ``preregistration.py`` should query ``Attendee``, while all functions in ``jobs.py`` should query ``Job``, etc.